gluten-core/src/main/scala/io/glutenproject/expression/ExpressionConverter.scala

```diff
@@ -17,19 +17,44 @@
 
 package io.glutenproject.expression
 
+import io.glutenproject.GlutenConfig
 import io.glutenproject.backendsapi.BackendsApiManager
 import io.glutenproject.execution.{NativeColumnarToRowExec, WholeStageTransformerExec}
-
+import io.glutenproject.expression.ConverterUtils.VELOX_EXPR_BLACKLIST
 import org.apache.spark.internal.Logging
 import org.apache.spark.sql.catalyst.expressions._
 import org.apache.spark.sql.execution._
 import org.apache.spark.sql.execution.exchange.BroadcastExchangeExec
 import org.apache.spark.sql.types.DecimalType
 
 object ExpressionConverter extends Logging {
-  def replaceWithExpressionTransformer(
-                                        expr: Expression,
-                                        attributeSeq: Seq[Attribute]): Expression =
+
+  def doValidate(blacklist: Map[String, String], expr: Expression): Boolean = {
```

Comment:
Could you add some notes/comments for this function to explain the purpose?

Reply:
Done